guides/hack/11-built-in-types/35-enum-class.md

```diff
@@ -130,7 +184,7 @@ function test1(): void {
 hhvm.hack.lang.enable_enum_classes=1
 ```
 
-To access the additional information added by `HH\MemberOf`, one has to change the function signature in the following way:
+To access the additional information added by `HH\MemberOf`, change the function signature in the following way:
```

Comment:
My first question when I got to here is "why would I want to do that?"; I had to scroll back up to the top.

I think the answer is something along the lines of "If you wanted to restrict it to members of the enum that implement the interface, rather than just anything that implements the interface, use `HH\MemberOf`"?